### DISCUSSION TITLE
Why repeater requires permissions for accessing location?

### DISCUSSION THREAD

**Discussion by theivankulikov** [Mygod/VPNHotspot#438] [General]
In my case location permission is disabled for all apps... And I was surprised that VPNHotspot trying to access device location. What is the reason? Why access to location is necessary?

**Comment by Mygod** [comment]
Unfortunately on certain Android versions this is required by the system. See row "Use repeater/temporary hotspot without location enabled" in [README](https://github.com/Mygod/VPNHotspot/blob/master/README.md) for more details.
